evap/evaluation/models.py

```diff
@@ -617,6 +617,12 @@ def can_results_page_be_seen_by(self, user):
             return self.is_user_responsible_or_contributor_or_delegate(user)
         return self.can_be_seen_by(user)
 
+    def can_reset_to_new(self):
+        """Is it possible to execute .reset_to_new() for this evaluation?"""
```

Comment:
I'd say this is probably fine without the comment, and generally, I'll vote to remove any comments that don't add value.